pkg/cmd/completion.go

```diff
@@ -33,6 +34,53 @@ func newCompletionCmd() *completionCmd {
 	return cc
 }
 
+var zsh_completion_instructions = `
```

Comment:
Couple small nits.

- I think convention in Golang is to use camel-case over snake-case
- Being that these are constants we can define them as such in a block:
```go
const (
    zshCompletionInstructions = `
...
`
    bashCompletionInstructionsMac = `
...
`
    bashCompletionInstructionsLinux = `
...
`
)
```

Reply:
+1, I think gofmt/the linter will complain about this.